crucible-llvm/src/Lang/Crucible/LLVM/Translation/Options.hs

```diff
@@ -0,0 +1,55 @@
+------------------------------------------------------------------------
+-- |
+-- Module           : Lang.Crucible.LLVM.MemModel.Options
+-- Description      : Definition of options that can be tweaked during LLVM translation
+-- Copyright        : (c) Galois, Inc 2021
+-- License          : BSD3
+-- Maintainer       : Rob Dockins <rdockins@galois.com>
+-- Stability        : provisional
+------------------------------------------------------------------------
+module Lang.Crucible.LLVM.Translation.Options
+  ( TranslationOptions(..)
+  , defaultTranslationOptions
+  , debugIntrinsicsTranslationOptions
+  ) where
+
+-- | This datatype encodes a variety of tweakable settings that apply during
+--   LLVM translation.
+data TranslationOptions
+  = TranslationOptions
+    { debugIntrinsics :: !Bool
+      -- ^ Should we translate @llvm.dbg@ intrinsic statements? The upside to
+      --   translating them is that they encode debugging information about a
+      --   program that can be useful for external clients. The downside is
+      --   that they can bloat the size of translated programs, despite being
+      --   no-ops during simulation.
+    , laxArith :: !Bool
```

Comment:
I think you've reversed the sense of the `laxArith` option in these comments.  When `False`, we apply strict arithmetic checking; when `True` we omit those checks.

I'm open to the idea that we should reverse the entire sense of this option ad call it something else; but the easiest thing to do would be to update these comments.

Reply:
Oops, that's a a silly oversight on my part. I've changed "Should we produce proof obligations" to "Should we omit proof obligations".